packages/jest-types/src/Config.ts

```diff
@@ -18,10 +18,17 @@ export type Path = string;
 export type Glob = string;
 
 export type HasteConfig = {
+  // Whether to hash files using SHA-1.
```

Comment:
If these use `/** comment */` syntax I think it shows up via intellisense? I might be wrong

Reply:
Ah, you're right. I'll fix those up

Reply:
Done